.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -1,6 +0,0 @@
-<component name="InspectionProjectProfileManager">
-  <profile version="1.0">
-    <option name="myName" value="Project Default" />
-    <inspection_tool class="TypeAnnotation" enabled="false" level="WARNING" enabled_by_default="false" />
-  </profile>
-</component>
```

Comment:
You might want to put this back

Reply:
Will put it back. It was somehow removed when I built the project first time.